app/views/support/Pricing.scala

```diff
@@ -21,10 +24,9 @@ object Pricing {
     def prettyPricing(currency: Currency) =
       s"${unsafePrice(currency).pretty} ${plan.billingPeriod.frequencyInMonths}"
 
-    def amount(currency: Currency) =
-      unsafePrice(currency).amount
+    def amount(currency: Currency) = unsafePrice(currency).amount
 
-    def prettyPricingForDiscountedPeriod(discountPromo: Promotion[PercentDiscount, Option, LandingPage], currency: Currency) = {
+    def prettyPricingForDiscountedPeriod[M[+A], A <: LandingPage](discountPromo: Promotion[PercentDiscount,M, A], currency: Currency) = {
```

Comment:
This *was* parametrised in Option, which meant that we could not use it with a Promotion once it had fetched its landing page. This is the same type parameterisation as used in the Promotion case class in membership-common. 

It's not obvious that we should split Promotions into Option of Landing Page and Identity of LandingPage.